applications/portfolioDrafts/portfolio/post.ts

```diff
@@ -0,0 +1,55 @@
+import { UpdateCommand } from "@aws-sdk/lib-dynamodb";
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { PortfolioStep } from ".././models/PortfolioStep";
+import { dynamodbClient } from ".././utils/dynamodb";
+
+const TABLE_NAME = process.env.ATAT_TABLE_NAME;
+const CLIENT = dynamodbClient;
+
+/**
+ * Handles requests from the API Gateway.
+ *
+ * @param event - The POST request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  if (!event.body) {
+    return { statusCode: 400, body: "" };
+  }
+
+  const portfolioId = event.pathParameters?.portfolioDraftId;
+
+  if (!portfolioId) {
+    return { statusCode: 400, body: "invalid request, you are missing the path parameter id:" };
+  }
```

Comment:
Is this even possible? Or can we assert that this is always there? Is it possible to omit a path parameter?

Reply:
a fine question